### ISSUE TITLE
Biases and BatchNorm not quantized as described in "AQD: Towards Accurate Quantized Object Detection"

### ISSUE THREAD

**Issue by ShechemKS** [aim-uofa/model-quantization-to-be-deleted-recently#6]
After reading the paper "AQD: Towards Accurate Quantized Object Detection", I have been using this repo to quantize an object detector. After reading the code, I realized that the biases of the convolutions (if it has biases) and batch normalization is not quantized. However, the paper "AQD: Towards Accurate Quantized Object Detection" states 
> We propose an Accurate Quantized object Detection (AQD) method to fully get rid of floating-point computation in each layer of the network, including convolutional layers, normalization layers and skip connections.

Specifically, I cannot find the code that corresponds to the equations given in section 3.2.2 of the paper. Am I missing something? How does that work in the code? Am I not using the correct keywords? (I have used the default ones provided: `keyword: ["debug", "dorefa", "lsq"]`). The biases don't seem to be quantized either.

Additionally, in the default configurations, the weights are quantized using the adaptive mode `var-mean` (i.e. the weights are normalized before being quantized, to my understanding). Is this also part of the method adopted in the paper, or should I disable this if I am to replicate those results?

**Comment by blueardour** [comment]
1. Whatever the quantization bitwidth for activation and weight, the bias is generally quantized into quite high bit ( INT32 or INT16).  Its quantization error is very small. Thus, in the AQD, I did not quantize the bias during training. Instead, I rounded it to integer just during evaluation. The accuracy mismatch was neglectable. Refer  https://github.com/aim-uofa/model-quantization-old/blob/bd7f6c261ef2ee9f51b7080a55c482b7ae726a3a/models/layers.py#L399

2. We emplyed  weight normalization (`var-mean`) for almost all the layers, except for the input and output layers. The `_C.MODEL.QUANTIZATION.wt_adaptive = "none"` option can enable/disable the manner for all layers. To enabe or disable customed layer, we provide `_C.MODEL.QUANTIZATION.policy ` option to configure the quantization hyperparameter in more fine-grain setting.
